docs/contributing/doing.md

```diff
@@ -0,0 +1,141 @@
+
+Here's a good resource for [Do's and Don'ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) when working with bugs, issues, and requests.
+
+[GitHub Labels](https://developer.github.com/v3/issues/labels/) are used to manage issues broadly considered work items grouped by color and prefixed by label name.
+
+## Creating branches
+
+Create a unique branch name tracking `origin/master` to perform your work.
+
+```bash
+git checkout -b <users/alias/feature-branch-name> origin/master
+```
+
+- Use all lower cased hyphen-delimited branch names for readability
+- Format as <users/alias/branch-name>. For example, users/jdoe/task-issue-title
+- Create as new branch for each work issue using the `-b` argument.
+- Commit messages should use lowercase imperative present tense. Use commands. Instead of “I added tests for ...” use “add tests for ...”
+
+## Pull requests
+
+When landing a pull request (PR), individuals must modify the original commit message to include comprehensive details. This detail along with other git history meta information deploys as part of the documentation changelog history. Changlogs are generated by [Lerna](https://lernajs.io/) using [Conventional commits](https://conventionalcommits.org/).
+
+PR comments should include the issue number per the [github closing issue keywords](https://help.github.com/articles/closing-issues-using-keywords/) documentation.
+
+One or more collaborators will automatically be assigned to review each PR based on areas of [ownership](https://github.com/Microsoft/fast-dna/blob/master/.github/CODEOWNERS).
+
+**`<type>`** is required to be at least one of the following:
+
+- chore: A change that does not impact distributed packages
+- feat: Adding, updating, or removing a feature
+- fix: A bug fix.
+
+**`<description>`** is required and speaks to what the user gets from this PR:
+
+- Be concise
+- Use lowercase
+- Use imperative, present tense (e.g. `add` not `adds`)
+- Do not end with a period. Period.
+- Avoid redundant words
+
+Accepted title format
+
+```comment
+<type>: <description>
+```
+
+Example titles
+
+```comment
+feat: add a border radius to MSFT button
+chore: add unit-tests for all components
+fix: update MSFT design system to use 3px border radius
+```
+
+### Formatting messages
+
+Optionally, include more detail in the message.
+
+`<body>` is used to provide clarity and context to the description.
+
+`<footer>` is used to provide additional metadata about the pull request such as issue fixing (fix #19, close #19, resolve #19). Additionally, is used to denote breaking changes.
```

Comment:
Per the other pull request comment please remove the "<" and ">" as these look like HTML tags

Reply:
Intended to be that way.  It's the way templates work on GitHub.